makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY = all all-force python-dependencies parts codes composed merge reason
+.PHONY: all all-force python-dependencies parts codes composed merge reason test
```

Comment:
Minor bugfix: My syntax on phony targets was wrong, so `make test` from command line was resulting in nothing happening. Fixed now.